packages/flutter_test/lib/src/window.dart

```diff
@@ -627,6 +627,16 @@ class TestFlutterView implements FlutterView {
   /// can only be set in a test environment to emulate different view
   /// configurations. A standard [FlutterView] is not mutable from the framework.
   ///
+  /// This property and [physicalGeometry] are dependent on one another. If both
```

Comment:
physicalGeometry -> physicalSize?

```suggestion
  /// This property and [physicalSize] are dependent on one another. If both
```

Reply:
Yup. Definitely that... My bad. Good catch!